tests/environment-install.bats

```diff
@@ -65,13 +65,24 @@ setup_file() {
   skip "remote environments handled in another phase"
 }
 
-@test "i?: confirmation message" {
+@test "i?: install confirmation message" {
   "$FLOX_CLI" init
   run "$FLOX_CLI" install hello
   assert_success
   assert_output --partial "✅ Installed 'hello' into 'test' environment."
 }
 
+@test "uninstall confirmation message" {
```

Comment:
```suggestion
@test "u?: uninstall confirmation message" {
```

I know we dont have a spec for this, we should and give it an id @ghudgins `u` for `uninstall` or `upgrade` or `update`, single letters won't last forever ;)

Reply:
CC @jennymahmoudi - no preference from my side. not sure what the right one to pick as the winner when there's conflicts. proceed with the one that helps us the most and we can validate its usability via testing